src/test/java/com/google/cloud/spanner/pgadapter/JdbcMockServerTest.java

```diff
@@ -3469,9 +3469,8 @@ public void testCursorSuccess() throws SQLException {
           getWireMessagesOfType(ExecuteMessage.class).stream()
               .filter(message -> !JDBC_STARTUP_STATEMENTS.contains(message.getSql()))
               .collect(Collectors.toList());
-      assertEquals(5, executeMessages.size());
-      assertEquals("", executeMessages.get(0).getName());
-      for (ExecuteMessage executeMessage : executeMessages.subList(1, executeMessages.size() - 1)) {
+      assertEquals(4, executeMessages.size());
```

Comment:
These changes are due to https://github.com/pgjdbc/pgjdbc/pull/3655/files, which now leads to BEGIN statements being sent using the simple query protocol instead of the extended query protocol.